### DISCUSSION TITLE
Filters, Ticket Data, things don't make sense here....

### DISCUSSION THREAD

**Discussion by quadcom** [osTicket/osTicket#5914] [Q&A] [answered]
I'm creating filters for tickets to auto assign/filter out admin type tickets.

Under the filter criteria the options for Ticket Data are;

Ticket/Issue Summary
Ticket/Issue Detail
Ticket/Priority Level

Priority Level is pretty simple but what about the other two? What are those referring to? How come there is no Ticket Subject as something to filter against? There is nothing talking about any of this in the docs.

**Comment by JediKev** [comment, marked as answer]
@quadcom 

Those fields are listed under **Admin Panel > Manage > Forms > Ticket Details**. By default these are named "Issue Summary", "Issue Details", and "Priority Level". You can change the Field Names to reflect whatever you'd like.

Issue Summary is the Subject of Tickets. Issue Details is the initial Message from the User when creating a Ticket. Priority Level is, well, the Priority Level.

We can take a look at our Docs to see where we can improve. Thanks for the feedback.

Cheers.
